task_processing/plugins/mesos/task_config.py

```diff
@@ -23,6 +27,17 @@ def valid_volumes(volumes):
     return (True, None)
 
 
+def _valid_constraints(constraints):
+    for constraint in constraints:
+        operator_name = constraint.operator
+        if not valid_constraint_operator_name(operator_name):
+            return (
```

Comment:
rather than exiting on the first failure, can you iterate over all the constraints to see which would have been invalid? That way the user gets to see all the failures up front.